### ISSUE TITLE
Feature request: How to merge two games that share a common point?

### ISSUE THREAD

**Issue by GeekyPeas** [DavidEGrayson/aoe2recfun#1]
Disclaimer: I feel this question should go on stack-overflow or reddit. However your library seems to come closest to the feature I am looking for. So decided to ask for a feature request to you directly.

Situation: We were playing a game and the game dropped. Then we restored the game. Now I have two recorded game: one before the drop and one after the restore.

Question: How can I merge the two records into one record?

Use-case: Actually I have many such recorded games that I have just manually renamed as part-1, part-2, part-3 etc... But in the end it becomes difficult to mannage. I wish I could merge them for some sanity.

p.s.: I also asked this question on [StackExchange Arqade](https://gaming.stackexchange.com/questions/398195/how-to-merge-two-recorded-games-in-aoe2-de-that-share-a-common-point)

I also asked this question on [Reddit AoE2](https://www.reddit.com/r/aoe2/comments/w17c9q/how_to_merge_two_recorded_games_in_aoe2_de_that/)

Prompted by [this comment](https://www.reddit.com/r/aoe2/comments/w17c9q/comment/igitjhj/?utm_source=share&utm_medium=web2x&context=3) I discovered another great github repository [aoc-mgz](https://github.com/happyleavesaoc/aoc-mgz). I have asked this question on [its github issues](https://github.com/happyleavesaoc/aoc-mgz/issues/86) as well.

**Comment by DavidEGrayson** [comment]
Looks like someone gave you an answer on Reddit :-)

**Comment by GeekyPeas** [comment]
I wish it worked. :-)

No, the Reddit method doesn't work as mentioned by the author on this [github issue](https://github.com/happyleavesaoc/aoc-mgz/issues/86#issuecomment-1186583442).

Do you have any thoughts on how this might be done?

(Also, can you please keep the issue open, unless you deem this is beyond the scope of your repository/ project?)

**Comment by DavidEGrayson** [comment]
Yeah, I think I should keep this feature request open, making it easier for other people to find it if they desire the same feature.

I don't think I'll have the energy to work on this any time soon, and it seems like such a niche feature that only a few people would need.  I think I had a restored game recently so maybe I can find that and figure out how to merge the recordings, but no promises.

By the way, your recordings only work in a specific version (or a few versions of DE), so even if you figure out how to merge them together, they still will become mostly useless in a few months.
